src/skai/model/inference_lib.py

```diff
@@ -11,6 +11,16 @@
 import tensorflow as tf
 
 
+def set_gpu_memory_growth() -> None:
+  gpus = tf.config.list_physical_devices('GPU')
+  if gpus:
+    try:
+      for gpu in gpus:
+        tf.config.experimental.set_memory_growth(gpu, True)
+    except RuntimeError as e:
+      print(e)
+set_gpu_memory_growth()
+
 import tensorflow_text  # pylint: disable=unused-import
```

Comment:
@panford Does this library have to be imported after the set_gpu_memory_growth() call? If not I will just move it up to where the rest of the imports are. Thanks!

Reply:
@jzxu They're not dependent. Please move feel to where other imports are